arelle/Cntlr.py

```diff
@@ -618,6 +618,10 @@ def workingOnlineOrInCache(self, url: str) -> bool:
         if os.path.exists(cacheFilepath):
             # The file exists in cache, we can proceed despite working offline
             return True
+        builtinCacheFilepath = self.webCache.urlToCacheFilepath(url, cacheDir=self.webCache.builtInCacheDir)
```

Comment:
It's not required for the taxonomy package schemas, since the scheme is hard coded and not provided by the user of package, but I updated anyway in case this gets reused.